singer-io/tap-impact
# v.0.0.4 Fix conversion_paths, increase decimal digits

# Description of change
Fix issue for `conversion_paths`. Increase JSON schema decimal digits to `multipleOf` = `1e-8`.

# Manual QA steps
Ran Discover, singer-check-tap, initial sync, and incremental sync to target-stitch
 
# Risks
Low - minor change, fixing decimal digits issue and conversion_paths issue.
 
# Rollback steps
Revert to v.0.0.3
